server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/UserRoutesTest.java

```diff
@@ -443,6 +485,9 @@ default void putShouldFailOnUnknownExceptionOnGetUpdateUser(UsersRepository user
         @RegisterExtension
         UserRoutesExtension extension = UserRoutesExtension.withVirtualHosting();
 
+        WithVirtualHosting() throws DomainListException {
```

Comment:
why do you need this constructor?

Reply:
Because I had to move the domain creation to the static methods withVirtualHosting and withoutVirtualHosting that would then throw a DomainListException. If I put the domain creation in the @BeforeEach setup, somehow it fails for the parameterized test, as it seems to try to create again the same domain without cleaning up between each run.

Not sure what is the best here... Letting it like that, or maybe put the creation in a try catch block, or defining the domain list in a BeforeAll as it never changes actually?

Reply:
I'm okay with it